src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_client_factory.py

```diff
@@ -6,19 +6,31 @@
 from azure.cli.core.commands.client_factory import get_mgmt_service_client
 
 
-def get_arm_service_client():
+def get_arm_service_client(cli_ctx):
     """Returns the client for managing ARM resources. """
     from azure.mgmt.resource import ResourceManagementClient
-    return get_mgmt_service_client(ResourceManagementClient)
+    return get_mgmt_service_client(cli_ctx, ResourceManagementClient)
 
 
-def get_storage_service_client():
+def get_storage_service_client(cli_ctx):
     """Returns the client for managing storage accounts. """
     from azure.cli.core.profiles import ResourceType
-    return get_mgmt_service_client(ResourceType.MGMT_STORAGE)
+    return get_mgmt_service_client(cli_ctx, ResourceType.MGMT_STORAGE)
 
 
-def get_acr_service_client(api_version=None):
+def get_acr_service_client(cli_ctx, api_version=None):
     """Returns the client for managing container registries. """
     from azure.mgmt.containerregistry import ContainerRegistryManagementClient
-    return get_mgmt_service_client(ContainerRegistryManagementClient, api_version=api_version)
+    return get_mgmt_service_client(cli_ctx, ContainerRegistryManagementClient, api_version=api_version)
+
+
+def cf_acr_registries(cli_ctx, *_):
```

Comment:
Will `*_` allow passing in a different `api_version`?

Reply:
No you'd have to add `api_version=None`.
Nothing uses `api_version` right now though in ACR.

Reply:
Yes we merged to use one api-version at one point. This is fine we can add it when we need to support multiple versions.